lab-anthony/app/components/app-gallery-text/app-gallery-text-directive.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const angular = require('angular');
+
+angular.module('galleryApp')
+.directive('appGalleryText', function() {
+  return {
+    restrict: 'E',
+    replace: true,
+    template: require('./app-gallery-text.html'),
+    scope: {
+      title: '@',
```

Comment:
You'll have an easier time if you pass the whole image object through, not each key on it individually.